app/controllers/ops_controller/ops_rbac.rb

```diff
@@ -1140,11 +1142,11 @@ def rbac_group_set_form_vars
   end
 
   def rbac_group_filter_expression_vars(field_expression, field_expression_table)
-    if @group && @group.entitlement && @group.entitlement[field_expression].kind_of?(MiqExpression)
-      @edit[:new][field_expression] = @group.entitlement[field_expression].exp
-    else
-      @edit[:new][field_expression] = nil
-    end
+    @edit[:new][field_expression] = if @group && @group.entitlement && @group.entitlement[field_expression].kind_of?(MiqExpression)
```

Comment:
here CC is surely right
`@group&.entitlement && @group.entitlement[field_expression].kind_of?`

Reply:
Fixed.